packages/react-ui-ag/src/Listings/MobileMapListing.js

```diff
@@ -1,316 +0,0 @@
-import React, { Component } from 'react'
-import PropTypes from 'prop-types'
-import themed from '@rentpath/react-themed'
-import classnames from 'classnames'
-import autobind from 'autobind-decorator'
-import get from 'lodash/get'
-import isEqual from 'lodash/isEqual'
-import LazyLoad, { forceCheck } from 'react-lazyload'
-import {
-  Button,
-  ToggleButton,
-  ListingComponents,
```

Comment:
are we able to also remove ListingComponents and ListingCell from react-ui-core?

Reply:
hmm ok it looks like ListingComponents is still being used by #967

Reply:
@m0rd3c4i @quantrells should we move ListingComponents and ListingCell into ag for redesign preperation as well?

Reply:
I'm not sure what our goal was/is with `react-ui`, honestly; it would seem to be even less defined with `react-ui-ag`. I'm on board with moving and building out the new stuff in ag.js.

Reply:
I agree that LisitingCell will also be a good candidate to move into ag. Its only used by the components in the `react-ui-ag/Lisitngs` folder.